CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ if(WIN32 AND NOT WINRT)
 	add_definitions(-DWITH_OFFICE_TRACING=0)
 endif(WIN32 AND NOT WINRT)
 
-add_subdirectory(folly)
```

Comment:
Hmm looks like something went off when I rebased with master. Didn't make any changes that weren't in the previous iter other than a comment update